src/app/pages/manage-organisation/findyouradministrator/findyouradministrator.component.scss

```diff
@@ -0,0 +1,7 @@
+li{
```

Comment:
Rename the file name and folder name to follow the current naming conventions 
ex:- forgot-password-success (include dashes between words)